db/migrate/20180626125655_rename_user_maintenance_zone_record.rb

```diff
@@ -0,0 +1,31 @@
+class RenameUserMaintenanceZoneRecord < ActiveRecord::Migration[5.0]
+  class Zone < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+
+    validates_presence_of   :name
```

Comment:
Is this necessary?

Reply:
If it is, it need to be multi-region aware.